_drafts/2017-04-18-tabulizer.Rmd

```diff
@@ -0,0 +1,205 @@
+
+ 1. Install the Java Development Kit, if you don't already have it on your system. (Note that the JDK is different from the Java Runtime Environment (JRE) that you almost certainly already have.) Details of how to do this vary a lot between platforms, so see the [README](https://github.com/ropensci/tabulizer#installation) for details.
+ 2. Install rJava using `install.packages("rJava")` and resolve any issues surrounding the `JAVA_HOME` environment variable that may need to be set before and/or after installing rJava. Again, see the [README](https://github.com/ropensci/tabulizer#installation) or [various question/answer pairs on StackOverflow](http://stackoverflow.com/search?q=%5Br%5D+rjava+install) for platform-specific instructions.
+ 3. Install tabulizer and tabulizerjars (the package containing the tabula java library) using your favorite GitHub package installer:
+
+```R
+library("ghit")
+ghit::install_github(c("ropensci/tabulizerjars","ropensci/tabulizer"))
+```
+
+This should work. If not, set `verbose = TRUE` in `ghit::install_github()` to identify the source of any issues. Some common problems are the dependency on the **png** package, which might need to be installed first. On Windows (depending on your version of R and how it was installed) may require setting `INSTALL_opts = "--no-multiarch"` in `ghit::install_github()`.
+
+If none of these steps work, scroll through [the GitHub issues page](https://github.com/ropensci/tabulizer/issues?utf8=%E2%9C%93&q=is%3Aissue) for anyone experiencing a similar problem and, if not resolved in any of those discussions, feel free to open an issue on GitHub describing your problem including the fully verbose output of `install_github()` and your `sessionInfo()`.
+
+# Unlocking elections data with tabulizer 
+
+Elections data are the bread and butter of a lot of quantitative political science research. Many researchers in my field need to know how many citizens voted and for whom in order to make sense of campaigns, election integrity, partisanship, and so forth. Yet a substantial amount election related data is locked in government-produced PDFs. Worse, national, state, and local governments have little to no standardization in the formatting of elections data, meaning even if one could figure out a computational strategy for extracting one kind of data about elections in one year from one state, that computational strategy would likely be useless in the same state in another year or in any other state. Elections provide a fantastic and highly visible example of "open" government data that's not really open or usable at all.
```

Comment:
s/substantial amount election related data/substantial amount of election related data